packages/toolpad-app/src/toolpad/AppEditor/PageEditor/RenderPanel/RenderPanel.tsx

```diff
@@ -1058,6 +1142,16 @@ export default function RenderPanel({ className }: RenderPanelProps) {
                 lastContainerChild.parentProp,
                 parent.parentIndex,
               );
+
+              if (isPageColumn(parent)) {
```

Comment:
i'll probably improve this function later to not have to repeat as much logic- should be done separately though as it relates to the whole editor and not just resizing